### DISCUSSION TITLE
Porting pi-base-tools notebook features to pi-base/web

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#835] [General]
I'd like to get me, @danflapjax, and @Not-Abram on a Zoom sometime to discuss taking some of the functionality of https://observablehq.com/@dinod123/pi-base-tools and implementing it on pi-Base proper. (Other community members are welcome to join.) I think this might be done in a couple steps:

- Spec'ing out specific features/functionality
- Implementing each feature as a standalone notebook
- Porting the notebook to a pi-Base component (with supporting libraries)

Would we have availability to chat Monday Nov 4 at 3pm Central? If not, when might be a good time to connect?

**Comment by danflapjax** [comment]
3 Central? That should be doable for me.

**Reply by StevenClontz** [reply]
Great! @Not-Abram?

**Reply by Not-Abram** [reply]
That works!

**Reply by StevenClontz** [reply]
Let's use https://southalabama.zoom.us/j/93833246214 - see y'all next week. (And see you this Friday for the regular community meeting as well if you can make it.)
